### DISCUSSION TITLE
GraphQL Query $where Clause Example

### DISCUSSION THREAD

**Discussion by freakyfriday** [payloadcms/payload#601] [Q&A] [answered]
Hello,

I'm trying to query using the $where clause but am receiving a typed error. I'm following the example at https://payloadcms.com/docs/queries/overview#graphql-find-queries but still can't figure out why the type conversion is failing.

Playground Query:

<img width="1206" alt="Screen Shot 2022-05-26 at 2 20 13 pm" src="https://user-images.githubusercontent.com/8381608/170415136-5bc9b800-2a06-4293-ae9c-9bd13f32e885.png">

Types:

<img width="1400" alt="Screen Shot 2022-05-26 at 2 22 31 pm" src="https://user-images.githubusercontent.com/8381608/170415394-d8af22a0-6140-45f1-ab77-701fafc58568.png">

Any assistance would be greatly appreciated :-)

Thanks

**Comment by freakyfriday** [comment, marked as answer]
If anyone is following I figured it out. I had to change the type of $where from JSON to Post_where:

<img width="739" alt="Screen Shot 2022-05-26 at 2 37 26 pm" src="https://user-images.githubusercontent.com/8381608/170416898-b479cca9-e67f-41cc-8b29-7d9cd3be4486.png">

**Reply by jmikrut** [reply]
@freakyfriday — I can do you one better.

While your syntax DOES work, the better way would be like this:

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/6651896/170497591-82b1b954-4604-4f1f-ba55-d86c9cf38b75.png">

No `$` needed. Just use the `where` argument!
